src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$calendar-day-with: 100px;
+
+.calendar {
+  width: $calendar-day-with * 7 + 6px;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  $days: (
+    mon: 0,
+    tue: 1,
+    wed: 2,
+    thu: 3,
+    fri: 4,
+    sat: 5,
+    sun: 6,
+  );
+
+  @each $day, $i in $days {
+    &--start-day-#{$day} {
+      .calendar__day:nth-child(1) {
+        margin-left: (
+          $calendar-day-with * ($i) + ($calendar-day-with * ($i) / 100)
+        );
+      }
+    }
+  }
+
+  @for $a from 28 through 31 {
+    &--month-length-#{$a} {
+      @for $i from $a through 31 {
+        .calendar__day:nth-child(#{$i + 1}) {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+
+    width: $calendar-day-with;
```

Comment:
Ensure that the variable `$calendar-day-with` is corrected to `$calendar-day-width` to maintain consistency and avoid potential errors.